bin/templates/scripts/cordova/lib/prepare.js

```diff
@@ -47,8 +47,8 @@ module.exports.prepare = function (cordovaProject) {
         return updateProject(self._config, self.locations);
     })
     .then(function () {
-        handleIcons(cordovaProject.projectConfig, self.root);
-        handleSplashScreens(cordovaProject.projectConfig, self.root);
+        handleIcons(cordovaProject.projectConfig, self.root, self.locations);
```

Comment:
@eymorale 
instead of passing self.locations, pass the directory to use
handleIcons(cordovaProject.projectConfig, self.locations.xcodeCordovaProj);
handleSplashScreens(cordovaProject.projectConfig, self.locations.xcodeCordovaProj);

then no need to do basename or create new variable originalName